extensions/ql-vscode/package.json

```diff
@@ -421,12 +421,14 @@
     "build-tasks": "^0.0.1",
     "chai": "^4.2.0",
     "css-loader": "~3.1.0",
-    "glob": "^7.1.4",
+    "glob": "7.0.0",
```

Comment:
is this downgrade necessary?

Reply:
oops. Things were really broken for a while and I was trying a bunch of things. I didn't mean to commit that change.